Client/Client.py

```diff
@@ -284,12 +299,20 @@ def getCutechessCommand(arguments, data, nps):
         '{0}-{1}'.format(data['test']['engine'], data['test']['dev']['name'])
     )
 
+    # In case devnetwork is specified append EvalFile option
```

Comment:
Change the comment to

``# Add evaluation weights for the Dev engine if specified ``

Reply:
Fixed.